lib/netsuite/records/item_fulfillment.rb

```diff
@@ -11,12 +11,12 @@ class ItemFulfillment
 
       fields :tran_date, :tran_id, :shipping_cost, :memo, :ship_company, :ship_attention, :ship_addr1,
         :ship_addr2, :ship_city, :ship_state, :ship_zip, :ship_phone, :ship_is_residential,
-        :ship_status, :last_modified_date, :created_date
+        :ship_status, :last_modified_date, :created_date, :created_from_ship_group
```

Comment:
@marianmosley could you extract all of the field additions into a separate PR for review?

Reply:
@marianmosley friendly reminder here!